tests/Frontend/CoreSpec.hs

```diff
@@ -251,15 +251,30 @@ postToForm (F g) = do
         (_, Just payload') <- runFailOnError $ postForm "" frm (\_ -> pure env)
         liftIO $ payload' `shouldBe` payload
 
-    it (show (typeOf g) <> " (process *in*valid form input)") $
-        pendingWith "not implemented."  -- FIXME
+    -- TODO: Make a separation
```

Comment:
"make a distinction"?  unless this is just for yourself (which i'm assuming), you should elaborate.